mono_repo/README.md

```diff
@@ -27,4 +27,116 @@ Available commands:
 Run "mono_repo help <command>" for more information about a command.
 ```
 
+### Example
+
+In a repository with a single package in the `sub_pkg` directory, add a
+`mono_pkg.yaml` file: 
+
+```yaml
+dart:
+ - dev
+
+stages:
+  - analyze:
+    - dartanalyzer
+    - dartfmt
+  - unit_test:
+    - test
+```
+
+Running `mono_repo travis` in the root yields two files:
+
+#### `.travis.yml`
```

Comment:
not sure showing the content provides a lot of value here? their contents should be transparent to the user?

and its just yet one more golden style test that has to be updated etc without providing much value